demisto_sdk/commands/common/tools.py

```diff
@@ -208,24 +209,29 @@ def get_remote_file(
     tag = tag.replace('origin/', '').replace('demisto/', '')
 
     github_path = urljoin(github_config.CONTENT_GITHUB_LINK, tag, full_file_path)
-
+    github_request = partial(requests.get, github_path, verify=False, timeout=10)
```

Comment:
don't like partial here. the way it was before was way clearer please revert this change.

Reply:
I think it's much more clearer 😉